pandas/tests/scalar/timestamp/test_timezones.py

```diff
@@ -327,9 +327,6 @@ def test_timestamp_constructor_near_dst_boundary(self):
             expected = Timestamp("2015-10-25 01:00").tz_localize(tz)
             assert result == expected
 
-            with pytest.raises(pytz.AmbiguousTimeError):
-                Timestamp("2015-10-25 02:00", tz=tz)
```

Comment:
Current implementation defaults to `fold==0` or `ambiguous==True`.

Reply:
This definitely seems like a breaking change, you are changing the semantics of `pytz`. `pytz` doesn't support `fold` and it's probably a bad idea to try to partially backport support for it here.

Reply:
Not really. All I'm doing is setting fold to 0 by default for Timestamp construction from ambiguous times. I based this decision on the behavior of pydatetime, which doesn't raise an error and just sets fold to 0.
```python
from datetime import datetime

dt = datetime(2019, 10, 27, 1, 30, 0, 0, tzinfo=gettz('Europe/London'))
dt.fold
OUT:
0
```
I also propose we consider reintroducing the error on ambiguous time construction later.

Reply:
This is very much the wrong way to use `pytz`. I strongly recommend reading through [`pytz`: The Fastest Footgun in the West](https://blog.ganssle.io/articles/2018/03/pytz-fastest-footgun.html) to understand why what you are doing is a breaking change. `pytz` *cannot* raise an error in the code you mentioned because nothing from `pytz` is ever invoked.

To be honest the semantics of `Timestamp` are a mess anyway and it's very hard to reason about them, but it's pretty clear that this changes a tested behavior that people could rely on, because you're changing the default behavior from "ambiguous times throw an error" to "ambiguous times default to `fold=0`".